features/support/hooks.js

```diff
@@ -34,8 +34,10 @@ module.exports = function () {
         });
     });
 
+/*
```

Comment:
@lbud same here. The TODO is apparently necessary now, since this.Around is gone. Am I breaking stuff here? Can this part be safely removed?